### PR TITLE
Find previous on shift + click 'find'

### LINKED ISSUE #369: shift-click find button should select previous ocurrence

### PR DESCRIPTION
**Pull request by callummr**
Fixes #369

### CODE REVIEW

**Comment by benogle** [comment]
Thanks! Can you write a test for this?

**Comment by callummr** [comment]
That I can, added and passing.

File: spec/find-view-spec.coffee [hunk 1]
```diff
@@ -494,6 +494,13 @@ describe 'FindView', ->
       expect(findView.resultCounter.text()).toEqual('3 of 6')
       expect(editor.getSelectedBufferRange()).toEqual [[2, 34], [2, 39]]
 
+    it "selects the previous match when the next match button is pressed while holding shift", ->
+      shiftClick = $.Event('click')
```
Comment (benogle) [inline spec/find-view-spec.coffee]: @nathansobo can you suggest a better way to create an event for this case?

Comment (nathansobo) [inline spec/find-view-spec.coffee]: If we're still in jQuery land this seems fine to me.

**Comment by benogle** [comment]
:heart: Thanks for this!

**Pull request merged**